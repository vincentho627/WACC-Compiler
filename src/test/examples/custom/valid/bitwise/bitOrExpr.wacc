# evaluating bit-wise or

# Output:
# 126
# 58
# 46

# Program:

begin
  int a = 42 ;
  int b = 84 ;
  int c = 56 ;
  int d = 6 ;
  println a | b ;
  println a | c ;
  println a | d
end
