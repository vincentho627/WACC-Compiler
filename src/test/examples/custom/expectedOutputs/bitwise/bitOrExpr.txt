126
58
46
